### PR TITLE
feat: Blog post about slack

### PR DESCRIPTION
**Pull request by JanCizmar**

### CODE REVIEW

**Comment by JanCizmar** [comment]
Please fix the issues if there's any. I am off.

File: blog/2024-06-13-slack-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+<!--truncate-->
+
+With Tolgee's new Slack integration, you can receive notifications of selected project events.
+
+You can set up notifying about the following events:
+- New keys are created
+- Base translation is changed
+- Translations are changed
+
```
Comment (marketachalupnikova) [inline blog/2024-06-13-slack-integration.mdx:36]:
```suggestion
## Tailored Notifications for Every Role

```

File: blog/2024-06-13-slack-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+When designing the integration, we were considering all the roles contributing to a project and
+so we made it configurable to enable each role in the project team to subscribe
```
Comment (marketachalupnikova) [inline blog/2024-06-13-slack-integration.mdx:38]:
```suggestion
we made it configurable to enable each role in the project team to subscribe
```

File: blog/2024-06-13-slack-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+you would like to see, e.g., `/tolgee subscribe <projectId> --events --on new_key`
+
+- **Translators** are usually interested in new key creations and base language updates.
+`/tolgee subscribe <projectId> --events --on new_key,base_changed`
+
+- **Proofreaders** are interested in target translation changes, e.g.,
+`/tolgee subscribe <projectId> --events --on translation_changed`
+
+## What's coming next?
+In the near future, you can expect additional events support for translation comments, batch operations, and more 🚀
```
Comment (marketachalupnikova) [inline blog/2024-06-13-slack-integration.mdx:53]:
```suggestion
We're not stopping here! Soon, you can expect additional events support for translation comments, batch operations, and more 🚀
```

File: blog/2024-06-13-slack-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
```
Comment (marketachalupnikova) [inline blog/2024-06-13-slack-integration.mdx:54]:
```suggestion
## TL,DR - Sync Your Team with Slack
```

File: blog/2024-06-13-slack-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+- The Slack integration provides notifications for project events.
+- It allows configurable alerts tailored to different roles, such as project managers, translators, and proofreaders.
+- Upcoming features will include support for additional events like translation comments and batch operations. 🚀
```
Comment (marketachalupnikova) [inline blog/2024-06-13-slack-integration.mdx:57]: Add banner at the end

File: blog/2024-06-13-slack-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    ),
+  }}
+/>
+
+We are excited to announce the release of the Slack integration. This integration allows you to receive
+notifications about your projects directly in your Slack workspace.
+
+<!--truncate-->
+
+With Tolgee's new Slack integration, you can receive notifications of selected project events.
```
Comment (marketachalupnikova) [inline blog/2024-06-13-slack-integration.mdx:30]: Add links to docs

**Pull request merged**